Xamarin.Forms.Platform.iOS/ContextActionCell.cs

```diff
@@ -653,7 +670,7 @@ public SelectGestureRecognizer() : base(Tapped)
 
 					var cell = table.CellAt(_lastPath) as ContextActionsCell;
 
-					return cell != null;
+					return cell != null && (!cell.IsOpen || !cell.IsTouchInDisabledButton(pos));
```

Comment:
Why check IsOpen here? This would seem to prevent selection of the cell when its open even if you were to tap on the left part which is still able to be selected.